### DISCUSSION TITLE
NetTcp   IsInitiating , IsTerminating

### DISCUSSION THREAD

**Discussion by ifpchan** [CoreWCF/CoreWCF#761] [Q&A]
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ServiceModel;
using CoreWCF;
using System.Runtime.Serialization;

  [CoreWCF.ServiceContract(  CallbackContract = typeof(ICorePerSessionCallBack),  SessionMode = CoreWCF.SessionMode.Required)]
    public interface ICorePerSession
    {

      [CoreWCF.OperationContract(IsOneWay = false, IsInitiating = true, IsTerminating = false)]
        string InInitiating();

        [CoreWCF.OperationContract(IsOneWay = true, IsInitiating = false, IsTerminating = true)]
        void InTerminating();

        [CoreWCF.OperationContract(IsOneWay = true, IsInitiating = false, IsTerminating = false)]
        void callHello(string hello);

Error : The type or namespace name 'IsInitiating' could not be found (are you missing a using directive or an assembly reference?)	
Error : The type or namespace name 'IsTerminating' could not be found (are you missing a using directive or an assembly reference?)	

Any one know how to using add IsInitiating, IsTerminating to OperationContract ?

**Comment by idunos** [comment]
We are facing the same issues.

**Comment by g7ed6e** [comment]
it looks like this is a duplicate of #715
